vhdbuilder/packer/vhd-image-builder-arm64-gen2.json

```diff
@@ -313,11 +313,6 @@
       "source": "parts/linux/cloud-init/artifacts/sshd_config_1604",
       "destination": "/home/packer/sshd_config_1604"
     },
-    {
-      "type": "file",
-      "source": "parts/linux/cloud-init/artifacts/sshd_config_1804_fips",
-      "destination": "/home/packer/sshd_config_1804_fips"
```

Comment:
why are we removing this?

Reply:
see https://github.com/Azure/AgentBaker/pull/4248#discussion_r1556492781